measurementWriter.go

```diff
@@ -1,14 +1,25 @@
 package main
 
-import "fmt"
+import (
+	"context"
+	"github.com/influxdata/influxdb-client-go/v2"
+	"github.com/influxdata/influxdb-client-go/v2/api"
+	"strconv"
+	"time"
+)
 
-type MeasurementPrintWriter struct {
+type MeasurementWriter struct {
+	writeApi    api.WriteAPIBlocking
 	measurement chan Measurement
 }
 
-func (mw *MeasurementPrintWriter) ReadMeasurement() error {
+func (mw *MeasurementWriter) StoreMeasurement() {
```

Comment:
Name of this method suggests that it will store a measurement once.

I think we should find better name for this method. I would be ok with just `go MeasurementWriter{...}.Start()`, but if you have better ideas go with it.